src/viam/media/video.py

```diff
@@ -159,10 +56,10 @@ class ViamImage:
     Provides the raw data and the mime type, as well as lazily loading and caching the PIL.Image representation.
     """
 
+    _height: Optional[int]
+    _width: Optional[int]
     _data: bytes
     _mime_type: CameraMimeType
-    _image: Optional[Image.Image] = None
-    _image_decoded = False
```

Comment:
We actually should probably keep these around.

We should attempt to have `width` and `height` available without the user calling `determine_image_dimension`. If we try to set these on `init`, then we actually move backwards in performance (since determining dimensions requires the use of PIL right now). So in order to keep our performance in line with what it used to be, we should only convert to PIL to get dimensions ONCE, and then keep that PIL.Image around so we don't have to always to the decoding.